src/Microsoft.AspNetCore.Mvc.ViewFeatures/Internal/ValidateAntiforgeryTokenAuthorizationFilter.cs

```diff
@@ -24,7 +22,7 @@ public ValidateAntiforgeryTokenAuthorizationFilter(IAntiforgery antiforgery, ILo
             }
 
             _antiforgery = antiforgery;
-            _logger = loggerFactory.CreateLogger<ValidateAntiforgeryTokenAuthorizationFilter>();
+            _logger = loggerFactory.CreateLogger(GetType());
```

Comment:
The `AutoValidateAntiforgeryTokenAuthorizationFilter` filter derives from this one and when logging happens from this derived filter it would still show up with base type.